spec/acceptance/2_realm_spec.rb

```diff
@@ -37,6 +37,7 @@ class { 'keycloak':
         offline_session_idle_timeout             => 2592000,
         offline_session_max_lifespan             => 5184000,
         offline_session_max_lifespan_enabled     => true,
+        user_managed_access_allowed              => true,
```

Comment:
Be good to add a validation this was set here:

https://github.com/treydock/puppet-module-keycloak/blob/e18d185b4ef4c71ba1f3d7be9eadf1aef1a75778/spec/acceptance/2_realm_spec.rb#L54

Also be useful to either move this `true` value to the update function and validate that is set there or just leave unset in update section and validate set to `false`.  Here:

https://github.com/treydock/puppet-module-keycloak/blob/e18d185b4ef4c71ba1f3d7be9eadf1aef1a75778/spec/acceptance/2_realm_spec.rb#L147

https://github.com/treydock/puppet-module-keycloak/blob/e18d185b4ef4c71ba1f3d7be9eadf1aef1a75778/spec/acceptance/2_realm_spec.rb#L191